Recommend container naming convention for runtime implementations (#1061)
serverlessworkflow/specification: dsl-reference.md

          image: fake-image
```

> [!NOTE]
> When a `container process` is executed, it is recommended that runtime implementations follow a predictable naming convention for the container name. This can improve monitoring, logging, and container lifecycle management.
>

> > We recommand using the following convention: `{workflow.name}-{uuid}.{workflow.namespace}-{task.name}`

Reviewer: ```suggestion
The ServerlessWorkflow specification recommends using the following convention: `{workflow.name}-{uuid}.{workflow.namespace}-{task.name}`
```

> > [!NOTE]
> > When a `container process` is executed, it is recommended that runtime implementations follow a predictable naming convention for the container name. This can improve monitoring, logging, and container lifecycle management.
> >
> The ServerlessWorkflow specification recommends using the following convention: `{workflow.name}-{uuid}.{workflow.namespace}-{task.name}`

Reviewer 2: ```suggestion
> [!NOTE]
> When a `container process` is executed, runtime implementations are recommended to follow a predictable naming convention for the container name. This can improve monitoring, logging, and container lifecycle management.
> The Serverless Workflow specification recommends using the following convention: `{workflow.name}-{uuid}.{workflow.namespace}-{task.name}`
```

Author: Done
